components/script/dom/element.rs

```diff
@@ -2260,10 +2274,14 @@ impl VirtualMethods for Element {
         }
 
         let doc = document_from_node(self);
+
+        doc.remove_focusable_element(self);
```

Comment:
We should only call this if we know that this element is has previously been added to the list.